static/metric-cardinality.html

```diff
@@ -0,0 +1,183 @@
+                                        <div id="now-30m" class="range-item">30 Mins</div>
+                                        <div id="now-12h" class="range-item">12 Hrs</div>
+                                        <div id="now-30d" class="range-item active">30 Days</div>
+                                    </div>
+                                    <div class="inner-range">
+                                        <div id="now-1h" class="range-item">1 Hr</div>
+                                        <div id="now-24h" class="range-item">24 Hrs</div>
+                                        <div id="now-90d" class="range-item">90 Days</div>
+                                    </div>
+                                    <hr>
+                                    <div class="dt-header">Custom Search <span id="reset-timepicker"
+                                            type="reset">Reset</span></div>
+                                    <div id="daterange-from">
+                                        <span id="dt-from-text">From</span> <br />
+                                        <input type="date" id="date-start" />
+                                        <input type="time" id="time-start" value="00:00" />
+                                    </div>
+                                    <div id="daterange-to">
+                                        <span id="dt-to-text">To</span> <br />
+                                        <input type="date" id="date-end">
+                                        <input type="time" id="time-end" value="00:00">
+                                    </div>
+                                    <div class="drp-buttons">
+                                        <button class="applyBtn btn btn-sm btn-primary" id="customrange-btn"
+                                            type="button">Apply</button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="mt-3 h-100 mb-2" id="tracing-container">
+                    <div class="border rounded p-4">
+                        <p class="caption">Total Unique Series</p>
+                        <div class="text-4xl font-bold" id="totalUniqueSeries">Loading...</div>
+                    </div>
+
+                    <p class="caption mt-4">Tag Keys with Most Series</p>
+                    <div id="tagKeysGrid" class="ag-theme-mycustomtheme"></div>
+                    
+                    <p class="caption mt-4">Tag Pairs with Most Series</p>
+                    <div id="tagPairsGrid" class="ag-theme-mycustomtheme"></div>
+                    
+                    <p class="caption mt-4">Tag Keys with Most Unique Values</p>
+                    <div id="tagKeysValuesGrid" class="ag-theme-mycustomtheme"></div>                    
+                </div>
```

Comment:
Add some margin below the last table.
<img width="687" alt="image" src="https://github.com/siglens/siglens/assets/71771131/f0d12eb0-928e-431a-a6d2-8f81ebdadc44">

Reply:
The margin is not being applied to the last table.